chia/harvester/harvester.py

```diff
@@ -196,12 +196,16 @@ def get_plots(self) -> tuple[list[dict[str, Any]], list[str], list[str]]:
                 param = prover.get_param()
                 if param.size_v1 is not None:
                     k = param.size_v1
+                    strength = uint8(0)
                 else:
+                    assert param.strength_v2 is not None
                     k = self.constants.PLOT_SIZE_V2
+                    strength = param.strength_v2
                 response_plots.append(
                     {
                         "filename": str(path),
                         "size": k,
```

Comment:
not related to this pr but why are we not working with Plot here instead of dict[str, Any]?

Reply:
these dictionaries are passed as results in the RPC protocol, as JSON. It could still go via the `Plot` class and have it convert to a JSON dict though.